boa/src/builtins/json.rs

```diff
@@ -1,10 +1,9 @@
 use crate::builtins::function::NativeFunctionData;
-use crate::builtins::object::{Object, ObjectKind, PROTOTYPE};
 /// The JSON Object
 /// <https://tc39.es/ecma262/#sec-json-object>
```

Comment:
Why do we have comments in an import? I think these should be module level documentation, and all imports should be together.